src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1012,6 +1012,22 @@ public static void onEmptyLeftClick(EntityPlayer player, ItemStack stack)
         MinecraftForge.EVENT_BUS.post(new PlayerInteractEvent.LeftClickEmpty(player, stack));
     }
 
+    public static net.minecraftforge.event.entity.player.CriticalHitEvent getCriticalHit(EntityPlayer player, Entity target, boolean vanillaCritical, float vanillaDefault)
```

Comment:
Is `vanillaDefault` a necessary parameter? It looks like it could be defined inside ForgeHooks based on `vanillaCritical`.
```java
float vanillaDefault = vanillaCritical ? 1.5f : 1f;
```